src/components/widgets/HeaderNotificationsDropdown/headerNotificationDropdown.less

```diff
@@ -0,0 +1,3 @@
+.dropdownMenu {
+  width: 380px !important;
```

Comment:
Because the original one was absolute... any suggestions on relative number which would be better? 25%, 30%? or something different? there might be a problem with percents on mobile devices, so define even `min-width`? How do you imagine this?

Reply:
I would use vw units for width and additionaly set min-width in px.

Reply:
OK